src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -121,7 +121,7 @@ private async Task Initialize(CancellationToken cancellationToken)
         int maxPeersCount = _networkConfig.ActivePeersMaxCount;
         Network.Metrics.PeerLimit = maxPeersCount;
 
-        _api.TxGossipPolicy.Policies.Add(new SyncedTxGossipPolicy(_api.SyncModeSelector));
+        _api.TxGossipPolicy.Policies.Add(new SyncedTxGossipPolicy(_api.SyncModeSelector, () => _api.BlockTree.IsProcessingBlock));
```

Comment:
It is weird that `IsProcessingBlock` is in BlockTree, can we remove it from there and only use the one in BlockchainProcessor?

Reply:
Also maybe `NotProcessingBlocksGossipPolicy` instead of delegate?

Reply:
> It is weird that IsProcessingBlock is in BlockTree,

Plumbing issue; BlockchainProcessor updates BlockTree; everything has reference to blocktree, not much has reference to the processor

Reply:
Moved to own policy attached to BlockchainProcessor